cmdstanpy/stanfit.py

```diff
@@ -312,6 +315,15 @@ def column_names(self) -> Tuple[str, ...]:
         """
         return self._column_names
 
+    @property
+    def stan_var_dims(self) -> Dict:
+        """
+        Dict mapping Stan program variable names to variable dimensions.
+        Scalar types have int value '1'.  Structured types have list of dims,
+        e.g.,  program variable ``vector[10] foo`` has entry ``('foo', [10])``.
+        """
+        return self._stan_var_dims
```

Comment:
this should return copy, otherwise

    print(a.stan_var_dims["a"]) --> 0
    res = a.stan_var_dims
    res["a"] = 3
    print(a.stan_var_dims["a"]) --> 3

Reply:
Also dims should be tuples

Reply:
looks like there's no such thing as read-only properties in Python. :-(
in which case, this is a problem all over the place, no?

Reply:
requires recoding everything with __get__ and __set__?  per https://howto.lintel.in/how-to-create-read-only-attributes-and-restrict-setting-attribute-values-on-object-in-python/

Reply:
Probably not, I think basic copy / deepcopy is enough